### ISSUE TITLE
[ADE 508 Audit]: Office of Section 508 Compliance: Claim or Appeal Status Website (44407): 508 Audit Report Findings Support

### ISSUE THREAD

**Issue by lakper** [department-of-veterans-affairs/va.gov-team#115220]
> [!NOTE]
> **QUICK SUMMARY:** _Parent ticket created to track Section 508 audit findings for the Claim or Appeal Status Website as part of Top Ten INTERnet Websites - June 2025 VASI._

## 508 Audit Office Scoring

**Overall Score:** `79`
- Critical: `1`
- High: `1`
- Medium: `0`
- Low: `3`
- Total: `79`

### About the 508 Office's Scoring

**A product with no defects has a score of 100**

- Each open 'Critical' defect lowers the score by 15.
- Each open 'High' defect lowers the score by 5.
- Each open 'Medium' defect lowers the score by 2.5.
- Each open 'Low' defect lowers the score by 1.
- Accessibility defects have no effect on the score.

### List of Defects:
The following examples are not and should not be interpreted as an exhaustive or complete list of all defects found in the software product. Examples were chosen to ease reproduction of the accessibility defect and to illustrate where multiple defects produce additional barriers to accessibility in combination. Stakeholders of the product should conduct internal testing to identify any additional instances of defects identified in this report. However, because these examples list the end-user experience of accessibility, defects identified may result from or be explained in terms of multiple criteria.
 
### 1. Link “Details” for the multiple claims is not unique.
Defect 13 | High | Created:

_WCAG 2.4.4 Link Purpose (In Context) | The purpose of each link can be determined from the link text alone or from the link text together with its programmatically determined link context, except where the purpose of the link would be ambiguous to users in general._

Check your claim, decision review, or appeal status -> Your claims, decision reviews, or appeals
Links are not uniquely identifiable. Multiple links feature the exact text without sufficient contextual cues to distinguish their purpose. Screen reader users do not understand which link to choose that will direct them to the relevant page. For instance, as a JAWS user in the “Your claims, decision reviews, or appeals” section on “Check your claim, decision review, or appeal status,” JAWS reads the same link text for two links as “Details for claim submitted on March 3, 2025,” for “Claims for disability compensation.“ Activating a link directs the user to the relevant page, and this does not allow the screen reader user to accurately identify and activate a specific link pertinent to the context. (New: 06/17/2025).

### 2. Top navigation menu links not structured properly
Defect 14 | Critical | Created:

_WCAG 2.1.1 Keyboard | All functionality of the content is operable through a keyboard interface without requiring specific timings for individual keystrokes_

Check your claim, decision review, or appeal status -> Your claims, decision reviews, or appeals
The top navigation menus lack proper structure. The popup menu cannot be dismissed using the keyboard. For example, when a JAWS user opens the submenu and tabs away from the popup menu, the menu remains opened and obstructs the content behind it. The ESC key does not dismiss the popup menu. The only way to close the menu is by using the mouse.
(New: 06/17/2025).

### 3. Duplicate Page Titles
Defect 15 | Low | Created:

_WCAG 2.4.2 Page Titled | Web pages have titles that describe topic or purpose._

Check your claim, decision review, or appeal status -> Your claims, decision reviews, or appeals
Throughout the web, duplicate page titles do not effectively convey the unique topic or purpose of the pages. For example, as a JAWS user in the “Your claims, decision reviews, or appeals” section on “Check your claim, decision review, or appeal status,” when the user activates the link “Details,” they are directed to the relevant page specific to a claim status; however, titles for the both pages are the same, and are recognized as “Status of March 3, 2025 Disability Compensation Claim | Veterans Affairs.” Users of assistive technology can become confused and disoriented when the titles lack contextual information. (New: 06/17/2025).

### Project Background

- Project Name: Top Ten INTERnet Websites - June 2025 VASI #: None  
- Audit Number: 44407 Audit Date: 6/11/2025
- Audit Scope: Claim or Appeal Status
- Product Type: Websites
- Project Status: Open
- Project Site: https://www.va.gov/claim-or-appeal-status
- Project Version: 06/11/2025
- Release Recommendation: No
- Release Recommendation Comments: Product is non-conformant
- Link to 508 Compliance Application: [508 Compliance Application](https://apps.gov.powerapps.us/play/e/default-e95f1b23-abaf-45ee-821d-b7ab251ab3bf/a/5f7da63e-384d-4ce4-8cc7-f3be6c60c363?tenantId=e95f1b23-abaf-45ee-821d-b7ab251ab3bf&sourcetime=2023-08-14%2017%3A50%3A55Z&RecordID=2989)

### Ticket Updates
- [ ] _Connect to an `Epic/Parent` or `Intake` (what body of work is this a part of?) If this is the `Epic/Parent`, connect related tickets as sub-issues._
- [ ] _If this is the `Epic/Parent` ticket, add label for `ADE 508 RECHECK`_
- [ ] _If this ticket is a `Child` ticket aka `sub-issue` please add the appropriate labels for `ADE 508 FLAG CRITICAL`;`ADE 508 FLAG HIGH`; `ADE 508 FLAG MEDIUM` or `ADE 508 FLAG LOW`_
- [ ] _Label with `Originator/Team` (product team or stakeholder requesting support; ADE if internal)_
- [ ] _Label date in the `Open Date` field_
- [ ] _Label with `Estimate` (level of effort expected for this ticket)_
- [ ] _Add `Assignee(s)` name(s) to ticket_ (include Lakwi)
- [ ] _Add `Assignee(s) name(s) to each task` they will complete via handle tag (if known)_
- [ ] _Select a `Priority Level`_
- [ ] _Update date in `Last Checked` field_
- [ ] _Label with `Actual` (level of effort it took to complete this ticket)_
- [ ] _Update date in_ `Closed Date`
- [ ] _Make sure the `ADE 508 RECHECK` label is on this ticket upon closing and that a follow-up date has been established and calendared._
